CodePush.m

```diff
@@ -362,11 +366,19 @@ - (void)savePendingUpdate:(NSString *)packageHash
 /*
  * This is native-side of the RemotePackage.download method
  */
-RCT_EXPORT_METHOD(downloadUpdate:(NSDictionary*)updatePackage
+RCT_EXPORT_METHOD(downloadUpdate:(NSDictionary*)immutableUpdatePackage
```

Comment:
I'd rather leave this as `updatePackage` and change the name of the mutable one to `mutablePackage`.